source/docs/index.markdown

```diff
@@ -40,7 +40,7 @@ The documentation covers beginner to advanced topics around the installation, se
     <div class='img-container'>
       <img src='/images/supported_brands/apple.png' />
```

Comment:
I think we should change this image also, for something that presents a mobile 📱